beetsplug/play.py

```diff
@@ -90,6 +90,9 @@ def play_music(self, lib, opts, args):
                 command_str = command_str.replace(ARGS_MARKER, opts.args)
             else:
                 command_str = u"{} {}".format(command_str, opts.args)
+        else:
+            if ARGS_MARKER in command_str:
```

Comment:
Also, could you please add a short comment here that explains why this is necessary? Posterity will thank you. :smiley: